continue-reading.html

```diff
@@ -4,10 +4,141 @@
+                  
+                    Lorem, ipsum dolor sit amet consectetur adipisicing elit. Minus tenetur consectetur consequatur reiciendis praesentium. Assumenda veritatis omnis magni veniam voluptatum inventore consequatur id impedit, nostrum dolorem vel at enim saepe.Lorem, ipsum dolor sit amet consectetur adipisicing elit. Minus tenetur consectetur consequatur reiciendis praesentium. Assumenda veritatis omnis magni veniam voluptatum inventore consequatur id impedit, nostrum dolorem vel at enim saepe.Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                    </p>
+                    <h3 class="totally"><strong>Lorem ipsum dolor, sit amet!</strong><br>
+                    </h3>
+                    <p class="totally_para">
+                  
+                  Lorem ipsum dolor sit amet, consectetur adipisicing elit. Possimus officia, molestias recusandae quisquam eveniet odio tenetur atque et laborum? Laborum sit suscipit officia repellat quis eius veritatis reprehenderit ullam est! Lorem ipsum dolor sit amet consectetur adipisicing elit. Nisi eos voluptatibus aut deleniti harum similique necessitatibus excepturi, voluptas nesciunt et? Animi eaque, dolorem eius provident quis magnam quas. Repellendus, veniam?
+                </p>
+                </div>
+          </div>
+  
+      
+          
+  
+          <!-- RIGHT SIDE POSTS BELOW -->
+  
+          <div id="right_container">
+              <div id="another_post">
+                  <div id="another_post_date" class="date">July 30</div>
+                  <h3 class="post_heading"><a href="blog-post.html"><strong>Just another post</strong></a></h3>
+                  <p class="another_post_para">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Earum, saepe temporibus. Beatae sint, sed excepturi unde veritatis fugit. Qui laboriosam impedit reprehenderit, placeat deleniti tempora animi minima nulla fugit in!</p>
+                  <div id="another_post_divider" class="divider"></div>
+                </div>
+        
+                <div id="and_another_post">
+                    <div id="and_another_date" class="date">July 28</div>
+                    <h3 class="post_heading"><a href="blog-post.html"><strong>And another post</strong></a></h3>
+                    <p class="another_post_para">Lorem ipsum dolor sit amet consectetur, adipisicing elit. Earum, saepe temporibus. Beatae sint, sed excepturi unde veritatis fugit. Qui laboriosam impedit reprehenderit, placeat deleniti tempora animi minima nulla fugit in!</p>
+                    <div id="and_another_divider" class="divider"></div>
+                </div>
+        
+                <div id="something_post">
+                    <h3 class="post_heading"><a href="blog-post.html"><strong>Something of note</strong></a></h3>

```

Comment:
For the bottom two you might not need the title to be a link since you are also using the "learn more" button under them